sdk/cosmos/azure-cosmos/azure/cosmos/aio/scripts.py

```diff
@@ -189,38 +196,41 @@ async def delete_stored_procedure(self, sproc, **kwargs):
     @distributed_trace_async
     async def execute_stored_procedure(
         self,
-        sproc,  # type: Union[str, Dict[str, Any]]
-        partition_key=None,  # type: Optional[str]
-        params=None,  # type: Optional[List[Any]]
-        enable_script_logging=None,  # type: Optional[bool]
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> Any
+        sproc: Union[str, Dict[str, Any]],
+        **kwargs: Any
+    ) -> Dict[str, Any]:
         """Execute a specified stored procedure.
 
         If the stored procedure does not already exist in the container, an exception is raised.
 
         :param sproc: The ID (name) or dict representing the stored procedure to be executed.
-        :param partition_key: Specifies the partition key to indicate which partition the sproc should execute on.
-        :param params: List of parameters to be passed to the stored procedure to be executed.
-        :param bool enable_script_logging: Enables or disables script logging for the current request.
-        :returns: Result of the executed stored procedure for the given parameters.
+        :type sproc: Union[str, Dict[str, Any]]
+        :keyword partition_key: Specifies the partition key to indicate which partition the stored procedure should
+            execute on.
+        :paramtype partition_key: Union[str, int, float, bool]
+        :keyword params: List of parameters to be passed to the stored procedure to be executed.
+        :paramtype params: List[Dict[str, Any]]
```

Comment:
We had discussed renaming this to parameters - are we gonna keep that for a separate PR?

Reply:
changed